cpp/src/gandiva/function_registry_arithmetic.cc

```diff
@@ -107,6 +107,13 @@ std::vector<NativeFunction> GetArithmeticFunctionRegistry() {
       BINARY_GENERIC_SAFE_NULL_IF_NULL(round, {}, int32, int32, int32),
       BINARY_GENERIC_SAFE_NULL_IF_NULL(round, {}, int64, int32, int64),
 
+      // bround functions
+      NativeFunction("bround", {}, DataTypeVector{float64()}, float64(),
+                     kResultNullIfNull, "bround_float64"),
+
+      NativeFunction("bround", {}, DataTypeVector{float64(), int32()}, float64(),
```

Comment:
Lets not add this in this PR. Please open a separate ticket for this and create a separate PR.

Reply:
Ok, I will open a new PR when as soon as this one is merged